genconfig.py

```diff
@@ -32,8 +32,8 @@ def dump_yaml(directory, filename, servers, port):
     arguments = parser.parse_args()
 
     if arguments.scylla:
-        dump_yaml(arguments.directory, 'scylla_servers.yaml', arguments.servers, scylla_port)
+        dump_yaml(arguments.directory, 'scylla_servers.yml', arguments.servers, scylla_port)
```

Comment:
@glommer can you please review?
How did this work so far? file name under prometheus dir are already with `yml` suffix